modules/viewing-the-service-logs-cli.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_admin/osd_logging/osd-accessing-the-service-logs.adoc
+// * rosa_cluster_admin/rosa_logging/rosa-accessing-the-service-logs.adoc
+
+:_content-type: PROCEDURE
+[id="viewing-the-service-logs-cli_{context}"]
+= Viewing the service logs by using the CLI
+
+You can view the service logs for 
+ifdef::openshift-dedicated[]
+{product-title} 
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA) 
+endif::openshift-rosa[]
+clusters by using the {cluster-manager} CLI (`ocm`).
+
+You can view the logs for a specific cluster or for all available clusters in your Red Hat organization. You can also filter the service logs, for example by severity or by log ID.
```

Comment:
Question, sir: Would it be an improvement to remove  ", for example" in this sentence?

Reply:
In my view, this comma helps in this sentence.